src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -104,6 +104,9 @@ static bool mpuSpi6000InitDone = false;
 
 static IO_t mpuSpi6000CsPin = IO_NONE;
 
+#define SPI_CLOCK_MPU6000_INITIALIZATION SPI_CLOCK_INITIALIZATON
+#define SPI_CLOCK_MPU6000_FAST           SPI_CLOCK_FAST
```

Comment:
Probably as a protest. What does FAST mean? It should be the lowest of two values: The maximum SPI clock frequency allowed by the processor (18 MHz for APB1 devices and 24 MHz for APB2 devices if I remember correctly) and the maximum allowed SPI frequency allowed by the device (20 MHz for MPU6000 for gyro reads). And why is the initialization speed set to 281 KHz when the MPU6000 allows 1 MHz? The way it is done is the single most confusing thing I had to deal with.